test/glific/flows_test.exs

```diff
@@ -532,7 +532,7 @@ defmodule Glific.FLowsTest do
       "An Interactive template does not exist",
       "A template could not be found in the flow",
       "Language is a required field",
-      "The next node after interactive should be wait for response",
+      "The next node after interactive (check 731b) should be wait for response",
```

Comment:
follow the similar format as other Node messages 

`The next node after interactive Node af33 should be wait for response`